data/mods/moderngen2/learnsets.ts

```diff
@@ -122,12 +122,12 @@ export const Learnsets: {[k: string]: ModdedLearnsetData} = {
 			worryseed: ["2L1"],
 		},
 		eventData: [
-			{generation: 3, level: 70, moves: ["2L1"], pokeball: "pokeball"},
```

Comment:
In `moderngen2` I've removed all the `moves: ["2L1"]` and `abilities: ["2L1"]` entries. I'm just assuming they don't do anything.